targets/ar71xx-generic/profiles.mk

```diff
@@ -193,10 +193,12 @@ endif
 
 ## D-Link
 
-# D-Link DIR-505 rev. A1
+# D-Link DIR-505 rev. A1/A2
 
 $(eval $(call GluonProfile,DIR505A1))
-$(eval $(call GluonModel,DIR505A1,dir-505-a1,d-link-dir-505-rev-a1))
+$(eval $(call GluonModel,DIR505A1,dir-505-a1-a2,d-link-dir-505-rev-a1))
```

Comment:
You can't just change the source filename from `dir-505-a1` to `dir-505-a1-a2`, the file won't be found like this.

I'll fix this up myself.